h/streamer.py

```diff
@@ -655,12 +620,82 @@ def should_send_annotation_event(socket, annotation, event_data):
     return True
 
 
+def process_message(reader, message):
+    """
+    Process a message from the queue.
+
+    Deserializes and distributes messages from the queue to the appropriate
+    handler functions.
+    """
+    if reader.topic == ANNOTATIONS_TOPIC:
+        broadcast_annotation_message(message, WebSocket.instances)
+    elif reader.topic == USER_TOPIC:
+        broadcast_session_change_message(message, WebSocket.instances)
```

Comment:
We certainly shouldn't crash, as that will requeue the message, which will result in another crash, etc, etc, up to `MAX_ATTEMPTS`, whatever that is at the moment...

I can certainly add a `log.warn` here.